src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final String PATTERN_FOR_DATE = "dd.MM.yyyy";
+    private static final String TITLE_FOR_REPORT = "Report for period ";
+    private static final String DASH_FOR_REPORT = " - ";
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final String STRING_FOR_SPLIT = " ";
```

Comment:
This is not mandatory to have constants for these 4 strings. But you should add constants for indexes 0-1-2-3 (NAME_INDEX, etc.)